lib/Bio/EnsEMBL/DataCheck/Checks/CheckAlphafoldEntries.pm

```diff
@@ -0,0 +1,63 @@
+=head1 LICENSE
+
+Copyright [2018-2022] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the 'License');
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an 'AS IS' BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+package Bio::EnsEMBL::DataCheck::Checks::CheckAlphafoldEntries;
+
+use warnings;
+use strict;
+
+use Moose;
+use Test::More;
+use Bio::EnsEMBL::DataCheck::Test::DataCheck;
+
+extends 'Bio::EnsEMBL::DataCheck::DbCheck';
+
+use constant {
+  NAME           => 'CheckAlphaFoldFormat',
+  DESCRIPTION    => 'Check records for alphafold import',
+  GROUPS         => ['protein_feature'],
+  DATACHECK_TYPE => 'critical',
+  DB_TYPES       => ['core'],
+  TABLES         => ['protein_feature'],
```

Comment:
Analysis is mentioned as well in the query, you may add `analysis` in the list then.

Reply:
Ok added